site/src/lib/api/blocks/slack.ts

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+
+import { mustGetEnvVar } from "../../../util/api";
+import { ExpandedBlockMetadata } from "../../blocks";
+
+export const notifySlackAboutBlock = async (
+  block: ExpandedBlockMetadata,
+  changeType: "publish" | "update",
+) => {
+  const webhookURL = mustGetEnvVar("SLACK_BLOCK_NOTIFICATION_WEBHOOK_URL");
```

Comment:
What happens if the env variable isn't there because it's being run locally, for example?

Reply:
Ciaran updated this here https://github.com/blockprotocol/blockprotocol/pull/614/commits/bfa10af1fefe2710d15f5bdd92fe5faa2db996ab

Now it just returns.